test/unit/api/threads.spec.ts

```diff
@@ -901,4 +905,106 @@ describe('Thread Tests', () => {
       expect(res2.body.error).to.be.equal(pinThreadErrors.MustBeAdmin);
     });
   });
+
```

Comment:
Should these tests be included?

Reply:
Unfortunately, until the `originURL = resolveURL(window.location.href);` bug is cleared up I can't run any tests. Kept these in the PR so the basic logic's laid down for the future.